sklearn/gaussian_process/kernels.py

```diff
@@ -2131,6 +2131,20 @@ class PairwiseKernel(Kernel):
         All entries of this dict (if any) are passed as keyword arguments to
         the pairwise kernel function.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_friedman2
+    >>> from sklearn.gaussian_process import GaussianProcessRegressor
+    >>> from sklearn.gaussian_process.kernels import PairwiseKernel
+
+    >>> X, y = make_friedman2(n_samples=500, noise=0, random_state=0)
+    >>> kernel = PairwiseKernel(metric='rbf')
+    >>> gpr = GaussianProcessRegressor(kernel=kernel,
+    ...         random_state=0).fit(X, y)
+    >>> gpr.score(X, y)
+    1.0
+    >>> gpr.predict(X[:2,:], return_std=True)
+    (array([781.9..., 518.4... ]), array([1.0...e-05, 1.0...e-05]))
```

Comment:
Thanks @steinfurt for your pull request.

```suggestion
    (array([781.9..., 518.4...]), array([1.0...e-05, 1.0...e-05]))
```
I hope this will fix the failing test.

Reply:
No, I don't think so, unfortunately. It seems the problem is more that in the second array, the values are too close to 1.0 * 10^{-5}, such that in some environments, one gets 9.999 * 10^{-6} which cannot be solved using the "..." in doctests. I have to think about this a bit, maybe change parameters or use something different. Any suggestions?

Reply:
You are right, and I can reproduce the issue on linux playing with the parameters.

> Any suggestions?

I think it is worth to use a different dataset: I've played with `noise` and `n_samples` and the std is really noisy around 1.  @glemaitre do you have any suggestions?

Reply:
I replaced the example with a corresponding classifier one using Iris. That should be fine.